plugins/modules/zos_operator_action_query.py

```diff
@@ -251,7 +279,28 @@ def run_module():
     try:
         new_params = parse_params(module.params)
 
-        cmd_result_a = execute_command("d r,a,s")
+        kwargs = {}
+
+        wait_s = new_params.get("wait_time_s")
+
+        zv = ZOAU_API_VERSION.split(".")
+        getit = False
+        if zv[0] > "1":
```

Comment:
I feel like this version check is going to result in a bug, so zoau 1.2.5.0 which is checked by this line of code elif zv[0] == "1" and zv[1] == "2" and zv[2] > "4": (which is correct) , but notice this is an if/elif so the very first if zv[0] > "1": would evaluate to true if the version of zoau is less than 1.2.5.0.

For example, if the version is 1.2.4.0, the first if will yield true and skip the last one elif zv[0] == "1" and zv[1] == "2" and zv[2] > "4": , all you need is the last check, so you can remove:

    if zv[0] > "1":
        getit = True
    elif zv[0] == "1" and zv[1] > "2":
        getit = True

also could you use something a bit more meaningful then getit as a variable name please, thank you.